requirements/optional-public.txt

```diff
@@ -5,7 +5,7 @@ https://github.com/cfpb/regulations-core/releases/download/1.2.6/regcore-1.2.6-p
 https://github.com/cfpb/regulations-site/releases/download/2.2.3/regulations-2.2.3-py2-none-any.whl
 https://github.com/cfpb/retirement/releases/download/0.6.1/retirement-0.6.1-py2-none-any.whl
 git+https://github.com/cfpb/ccdb5-api.git@v1.0.5#egg=ccdb5-api
-git+https://github.com/cfpb/ccdb5-ui.git@v1.0.9#egg=ccdb5_ui
+git+https://github.com/cfpb/ccdb5-ui.git@v1.0.10#egg=ccdb5_ui
 https://github.com/cfpb/django-hud/releases/download/1.4.3/django_hud-1.4.3-py2-none-any.whl
 https://github.com/cfpb/django-college-costs-comparison/releases/download/1.5.3/comparisontool-1.5.3-py2-none-any.whl
-https://github.com/cfpb/teachers-digital-platform/releases/download/0.5.16/teachers_digital_platform-0.5.16-py2-none-any.whl
\ No newline at end of file
+https://github.com/cfpb/teachers-digital-platform/releases/download/0.5.16/teachers_digital_platform-0.5.16-py2-none-any.whl
```

Comment:
I wonder why git sees this line as changing?

Reply:
I edited the file in Github, so my guess is the trailing newline got changed from Mac to Linux